### ISSUE TITLE
Problem with TransferDuplicatePokemonOnCapture latest compiled update

### ISSUE THREAD

**Issue by Whistler212** [NoxxDev/NecroBot#698]
Latest Update ignore TransferDuplicatePokemonOnCapture=true, bot didnt transfer immediately
### Expected Behavior
TransferDuplicatePokemonOnCapture=true
Pokemon on transfer category immediately transferred on catch high CP/IV

### Actual Behavior
Pokemon on transfer category didnt transferred on catch high CP/IV

### Your config.json
```
your config here
```

### Other Information
Version:
Windows Version:
What is affected (catch, transfer, evolve, recycle, snipe, powerup/levelup, other):

**Comment by tws101** [comment]
What is you keep min duplicate pokemon value in the config?

We need to see your config

**Comment by samuraitruong** [comment]
No, it maybe the feature to ignore transfer pokemon during walking to snipe location.

**Comment by Whistler212** [comment]
yes that is right, it happen only during walking snipe, can i change option to enable or disable ignore transfer pokemon during walking to snipe location?

**Comment by samuraitruong** [comment]
During snipe it should turn off for saving walking time

Get Outlook for Android<https://aka.ms/ghei36>



On Tue, Aug 30, 2016 at 9:32 AM +1000, "Whistler212" <notifications@github.com<mailto:notifications@github.com>> wrote:


yes that is right, it happen only during walking snipe, can i change option to enable or disable ignore transfer pokemon during walking to snipe location?

**Comment by Whistler212** [comment]
but it makes lot catchflee if i set speedup to 60, if i set 50, its run smoothly

**Comment by samuraitruong** [comment]
catchflee even happen to pokemon go app. try to catch when you driving you will see that issue.
